### PR TITLE
Add copied indicator

### PR DESCRIPTION
**Pull request by Steve0Greatness**
Buttons that don't appear to trigger anything could be confusing to end users.

### CODE REVIEW

**Review by CocoTheMii — COMMENTED** [review]
This is a good start, but the "copied" text should probably appear closer to the actual button.

As it stands, the button and the notification are currently on opposite ends of the screen, making it easy to miss.

**Comment by Steve0Greatness** [comment]
Would a better place be under the sidebar?

**Comment by lopste** [comment]
I think the button text being changed to “Copied!” for a bit would be best, as that’s where the user would be looking.

**Comment by Steve0Greatness** [comment]
Something like this?

https://user-images.githubusercontent.com/75220768/235812222-51ffbbb8-d4ef-4241-a481-6c18f46d8b6b.mp4

**Comment by CocoTheMii** [comment]
That looks good— what are these bugs you're describing?

**Comment by Steve0Greatness** [comment]
> That looks good— what are these bugs you're describing?

It may just be Microsoft's screen recorder, but - some of the buttons look a little delayed in the video.

**Comment by CocoTheMii** [comment]
I didn't notice any lag when testing the buttons. However, for some reason, only one input can say "Copied!" at once. I'm not familiar enough with JavaScript to figure out what's causing this.

I did make one change: I noticed that you had created a new attribute, `data-bbcode`, for BBcode values in each input. I removed this and replaced references to it with the existing `value` attribute, as it appears to contain the same information.

**Comment by Steve0Greatness** [comment]
> I did make one change: I noticed that you had created a new attribute, `data-bbcode`, for BBcode values in each input. I removed this and replaced references to it with the existing `value` attribute, as it appears to contain the same information.

That's actually used to save the BBCode that would normally be contained in the input box. I did this since `Copied!` is shown in the Textbox using the `value` attribute. This allows for using the copy button to not be affected by the text changing to `Copied!`, which if the user clicked the button 2 times, it would copy `Copied!` instead of the BBCode value. It's also used inorder to revert `value` back to normal after `Copied!` has been shown.

TL;DR : That's very important, I'd suggest it be reverted.

**Comment by Steve0Greatness** [comment]
> I didn't notice any lag when testing the buttons. However, for some reason, only one input can say "Copied!" at once. I'm not familiar enough with JavaScript to figure out what's causing this.

I believe the reason for this may have something to do with it using JS's builtin `settimeout` function. I don't fully understand the way that `settimeout` works yet, but that is most-likely the breakpoint.

**Comment by CocoTheMii** [comment]
> That's actually used to save the BBCode that would normally be contained in the input box. I did this since `Copied!` is shown in the Textbox using the `value` attribute. This allows for using the copy button to not be affected by the text changing to `Copied!`, which if the user clicked the button 2 times, it would copy `Copied!` instead of the BBCode value. It's also used inorder to revert `value` back to normal after `Copied!` has been shown.

I considered this before making the change, but I [tested it on GitHub Pages](https://cocothemii.github.io/forumoji) and it still works properly as far as I can tell. The text changing to "Copied!" seems to have no effect on the actual `value` attribute.

**Comment by Steve0Greatness** [comment]
Fixed the bug that made only one textbox fill at a time.
